scripts/docs/generate-docs-site.js

```diff
@@ -39,6 +39,7 @@ function copyApiReference(playbook, docsSrcPath) {
     // Checkout the repo to a temporary directory
     const projectRootDir = process.cwd();
     const gitTempDir = `${docsSrcPath}/_temp`;
+    runCmdSync(`rm -rf ${gitTempDir}`);
```

Comment:
I suggest

```suggestion
   fse.removeSync(gitTempDir);
```

to keep the script cross-platform.

(for consistency, we should probably also replace `mkdir` with `fse.mkdirSync()`)

Reply:
replaced with `fse.emptyDirSync`